libs/shared-config/src/lib/config.ts

```diff
@@ -368,6 +368,9 @@ export const MENTORSHIP_MATCH_STATUS_LABELS: any = {
   INVALIDATED_AS_OTHER_MENTOR_ACCEPTED: 'Cancelled',
 }
 
+export const NEST_API_URL = process.env.NX_NEST_API_URL
```

Comment:
The Nest URL is now defined here and in `libs/data-access/src/lib/graphql-client.ts`. Can we define the source-of-truth variable here and derive it elsewhere?

Reply:
Nice - I missed that before. 😎